packages/release/release.spec

```diff
@@ -101,6 +104,9 @@ install -p -m 0644 %{S:11} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
 install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/wicked/ifconfig
 install -p -m 0644 %{S:1000} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/wicked/ifconfig
 
+install -d %{buildroot}%{_cross_libdir}/repart.d
+install -p -m 0644 %{S:96} %{buildroot}%{_cross_libdir}/repart.d/80-local.conf
```

Comment:
Why is this a drop-in, and not part of the unit itself?

Reply:
`systemd-repart` looks for its config files in `/usr/lib/repart.d`, like how`systemd-tmpfiles` looks in `/usr/lib/tmpfiles.d`.